src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,65 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer manufacturerKia = new Manufacturer();
+        manufacturerKia.setName("Kia");
+        manufacturerKia.setCountry("South Korea");
+
+        Manufacturer manufacturerMercedes = new Manufacturer();
+        manufacturerMercedes.setName("Mercedes-Benz");
+        manufacturerMercedes.setCountry("Germany");
+
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        manufacturerService.create(manufacturerKia);
+        manufacturerService.create(manufacturerMercedes);
+
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+
+        List<Driver> driversForKia = new ArrayList<>();
+        driversForKia.add(driverService.get(1L));
+        driversForKia.add(driverService.get(2L));
+
+        List<Driver> driversForMercedes = new ArrayList<>();
+        driversForMercedes.add(driverService.get(3L));
+        driversForMercedes.add(driverService.get(4L));
+        driversForMercedes.add(driverService.get(5L));
```

Comment:
It's better not to hardcode the id. Use getId method
Check the same problems in the following code